core/debugger/local_debugger.cpp

```diff
@@ -84,7 +84,7 @@ struct LocalDebugger::ScriptsProfiler {
 			}
 		}
 
-		SortArray<ScriptLanguage::ProfilingInfo, ProfileInfoSort> sort;
+		SortArray<ScriptLanguage::ProfilingInfo, ProfileInfoSort> sort{};
```

Comment:
I'm not sure why it wants to initialize `SortArray` like this but not `Vector` and `TypedArray`. The only field in `SortArray` seems to be the `Comparator` template instance?